From 91c7eb146ba7d08cfbb015beaf601304872f7ca9 Mon Sep 17 00:00:00 2001
From: Bartosz Majsak <bartosz.majsak@gmail.com>
Date: Wed, 15 Dec 2021 11:06:10 +0100
Subject: chore: aligns tabs

---
 .editorconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.editorconfig b/.editorconfig
index cb130e1..89d1334 100644
--- a/.editorconfig
+++ b/.editorconfig
@@ -16,4 +16,4 @@ trim_trailing_whitespace = false
 
 [{*.go, Makefile}]
 indent_style = tab
-indent_size = 4
+indent_size = 2
-- 
2.31.1

