From 54528864028f5eaf5b6ba4a8a4e9b82b28bc017c Mon Sep 17 00:00:00 2001
From: bartoszmajsak <bartosz.majsak@gmail.com>
Date: Tue, 25 Jan 2022 15:55:32 +0100
Subject: bla README

---
 README.adoc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.adoc b/README.adoc
index f30ef87..d65fec5 100644
--- a/README.adoc
+++ b/README.adoc
@@ -4,7 +4,7 @@ Basic structure for cli-based golang projects.
 
 == Code
 
-- no-code
+- Folder structure
 
 == Setup
 
-- 
2.31.1

